packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -474,7 +570,11 @@ export default class DropdownCore extends React.Component<
 
         const itemRole = this.getItemRole();
 
-        let focusCounter = 0;
+        // If SearchTextInput is visible, we consider that as the first
+        // focusable item.
+        const showSearchTextInput =
+            !!onSearchTextChanged && typeof searchText === "string";
+        let focusCounter = showSearchTextInput ? 1 : 0;
```

Comment:
Feels like a smell that this is a `let` rather than `const`. No change requested, but I suspect we can tidy this code up in a future change.

Reply:
/cc @kevinbarabash @jandrade